bears/java/JavaPMDBear.py

```diff
@@ -50,6 +51,8 @@ def create_arguments(filename, file, config_file,
                          allow_unnecessary_code: bool = False,
                          allow_unused_code: bool = False):
         """
+        :param rule_set_file:
```

Comment:
Could you rename this `pmd_config` as that appears to be a common name.  c.f. https://github.com/coala/coala/issues/4072